integration_test/third_party_apps_data/applications/couchdb/expected_logs.yaml

```diff
@@ -0,0 +1,15 @@
+log_entries:
+- log_name: "couchdb"
+  field_matchers:
+    httpRequest.remoteIp: "127.0.0.1"
+    httpRequest.requestMethod: "GET"
+    httpRequest.responseSize: "1"
+    httpRequest.serverIp: "localhost:5984"
+    httpRequest.status: 404
+    jsonPayload.host: "127.0.0.1"
+    jsonPayload.message: "admin GET /forbidden 404 ok 1"
+    jsonPayload.node: "couchdb"
+    jsonPayload.path: "/forbidden"
+    jsonPayload.pid: "0.25387.37"
```

Comment:
This is not likely to be consistent between test runs, probably do not try to expect PID at all.

Reply:
updated